front/cypress/e2e/native_survey_page_action.cy.ts

```diff
@@ -122,12 +122,24 @@ describe('Native survey project page actions', () => {
     cy.get('[type="checkbox"]').check();
     cy.setLoginCookie(userEmail, userPassword);
     cy.visit(`/projects/${projectSlugTimeline}`);
-    cy.contains('Take the survey').click({ force: true });
+    cy.get('[data-testid="e2e-project-survey-button"]').click({ force: true });
```

Comment:
@EdwinKato I notice you tend to add comments in your e2e tests for clarity. I wasn't quite sure what was best practice - should I add more comments in my e2e tests?

Reply:
Good question 🤔 .I have no heavy inclination on this. I add comments because there is a similar pattern in the codebase but also to make it easy to follow the tests. Unlike normal code where reading the code gives insight into what is happening, cypress tests are a little bit different because you would need to look for the selectors in the codebase to understand what is happening.

Reply:
Ahh ok yes that's a good point. I'll add some comments then as well :+1: